doc/design/manifest.md

```diff
@@ -108,10 +108,6 @@ A key definition is:
 
 ```json
 "KEYID": {
-    "keyid_hash_algorithms": [
```

Comment:
I think both `sha256/sha512` and can be supported in the current version. see: https://github.com/pingcap-incubator/tiup/pull/263/files#diff-7e28c891f69ae43e5142e9e8c085c0deR28

Reply:
I don't think we need `keyid_hash_algorithms` in any case because it is not role dependent. We currently only read the sha256 hash, but we can still have multiple hashes so support sha512 easily in the future.

Reply:
I added back in the line of the spec about sha256/sha512, but left `keyid_hash_algorithms` out.